#168844244:apply feedback to ui

kosekaku opened the pull request:
What does this PR do?

    fix ui design issues

Description of Task to be completed

    make a sign in and descriptive image flow on a single column
    make a signup and descriptive image flow on a single column
    add more content to the view specific page
    add a consistent font family
    reduce image size for aside contents
    replace X letter for closing window page with a close window icon
    and other designs

How should this be manually tested?

    clone this repo
    cd to teamwork folder
    git checkout bg-fix-ui-feedback-168844244
    visit various html pages

Any background context you want to provide?

    N/A

** What are the relevant pivotal tracker stories?**

    #168844244

*screenshots* 
   
  >![image](https://user-images.githubusercontent.com/18526493/66026402-10b81580-e4f9-11e9-9b13-c4ac6ac6ac17.png)
  
>![image](https://user-images.githubusercontent.com/18526493/66026283-cc2c7a00-e4f8-11e9-885e-54c657b3a1e7.png)
    
    Latest Screenshots
>![image](https://user-images.githubusercontent.com/18526493/66044584-28f05a80-e522-11e9-83c7-61d4f7b887eb.png)
>![image](https://user-images.githubusercontent.com/18526493/66044618-432a3880-e522-11e9-8a8d-758c06a715ac.png)

kosekaku committed:
bg(fix ui design):fix ui design as per LFA feedback

make signin and descriptive image flow on single column
make signup and descriptive image flow on signle column
add more content to the view specific page
add consistent font family
reduce image size for aside contents
replace X leter for closing window page with a close window icon
and other designs

[Start #168844244]

kosekaku committed:
bg(ui design): fix some ui changes [Start #168844244]

kosekaku committed:
bg(ui design): fix some ui changes [Start #168844244]

kosekaku committed:
ft(ui design):change ui navigation search box

redesign navigation search bar

[Start #168844244]

YvesIraguha reviewed (changes requested):
Hello @itsUk45, Thanks for the good work of implementing feedback on the login/sign up page. Now when you navigate from login/sign up page to all articles page, they look like two different applications.  The reason why is on sign up page, you have a dark transparent background with an image in the background. However, on the other pages, you have a grey background. You can either keep a dark transparent background across the application, or you can have a clear background across the whole application. Moreover, the buttons are styled differently. Try to have consistency in the application. If you choose to have a button of this style 
![Screen Shot 2019-10-02 at 10 10 01](https://user-images.githubusercontent.com/34691070/66028199-dc465880-e4fc-11e9-900e-64ca43953839.png) 
please keep the same style across the application, and if you choose to go with these 
![Screen Shot 2019-10-02 at 10 11 36](https://user-images.githubusercontent.com/34691070/66028279-0dbf2400-e4fd-11e9-844c-980aca57e4a4.png)
keep using it. So, use consistent styles across the application. 
Another point is on this picture, the size of an icon is small compared to the text, please ensure that icons and text are of the same size
![Screen Shot 2019-10-02 at 10 13 30](https://user-images.githubusercontent.com/34691070/66028452-6e4e6100-e4fd-11e9-8d6b-21dc858cb58d.png)

And the title should be aligned with the rest of the article card content on this picture. 
![Screen Shot 2019-10-02 at 10 17 20](https://user-images.githubusercontent.com/34691070/66028695-ed439980-e4fd-11e9-9a94-859d0dabff7d.png)

Lastly, the pull request should be pointing to `gh-pages` not `develop`.

kosekaku committed:
bg(ui design issues):apply changes to ui design

apply background consistency
apply button consistency
redesign aside icon and text to have same size
align article's title and body properly

[Start #168844244]
